widgets/GCSMultiFiles-batchsink.json

```diff
@@ -0,0 +1,138 @@
+
+{
+  "metadata": {
+    "spec-version": "1.5"
+  },
+  "display-name": "Multi Files Google Cloud Storage Sink",
+  "configuration-groups": [
+    {
+      "label" : "Basic",
+      "properties" : [
+        {
+          "widget-type": "textbox",
+          "label": "Reference Name",
+          "name": "referenceName"
+        }
+      ]
+    },
+    {
+      "label" : "Credentials",
+      "properties" : [
+        {
+          "widget-type": "textbox",
+          "label": "Service Account File Path",
+          "name": "serviceFilePath",
+          "widget-attributes" : {
+            "placeholder": "Path to the service account file on local file system"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Project ID",
```

Comment:
this should be in the basic section. Please follow the structure, defaults, and placeholders in GCS-batchsink.json so that there is consistency across the plugins.